README.md

```diff
@@ -1 +1,31 @@
-# iOS-interview
\ No newline at end of file
+# GIPHY iOS Code Challenge
+
+Please look at the challenge closely and make sure to cover all of the requirements.  The goal is to build a product that would reflect an app ready for production, but make sure to consider the tradeoffs necessary to complete the app in 90 minutes.
+
+## ⭐️ Technical Requirements
+- Swift is required
+- No SwiftUI, we love SwiftUI but we need to make sure you understand UIKit
```

Comment:
I know you said you're not comfortable with SwiftUI yourself but would you be comfortable if someone used it? even if you're not an expert with it

Reply:
You know, I thought about this, and I would be willing to let someone use SwiftUI if they could justify the benefit and tradeoff.  

I am happy to pair with someone using SwiftUI, I do know SwiftUI I am just not super fluent with it yet.